enhancements/network/on-prem-dual-stack-vips.md

```diff
@@ -258,6 +265,21 @@ will be maintained. We will not (and cannot) automatically add new VIPs to a
 cluster on upgrade. If a deployer of an older dual stack cluster wants the new
 VIP functionality that will have to be a separate operation from upgrade.
 
+MCO takes care updating the new fields (`apiServerInternalIPs` &
+`ingressInternalIPs`) and old fields (`apiServerInternalIP` &
+`ingressInternalIP`) in [openshift/api](#openshiftapi) to have a consistent API
```

Comment:
I didn't fully understand this. Did you mean to say that MCO will keep up-to date these new fields from openshift/api (cluster infrastructure CRD) into MCO controllerconfig ? Note that controllerconfig is MCO internal.

Reply:
@sinnykumari: What I mean is that MCO will keep the values between the old fields (`apiServerInternalIP` & `ingressInternalIP`) and the new fields (`apiServerInternalIPs` & `ingressInternalIPs`) consistent. So users consuming the old API fields (`apiServerInternalIP` & `ingressInternalIP`) will get "updated" values. The rules of the update logic between these fields are described in the table later and will be implemented by @cybertron in https://github.com/openshift/machine-config-operator/pull/3269
The PR for the API changes is https://github.com/openshift/api/pull/1152

Reply:
got it, thanks for clarifying